examples/htmlserver/source/app.d

```diff
@@ -1,10 +1,11 @@
 import diet.html;
 import vibe.http.server;
+import vibe.internal.interfaceproxy : asInterface;
 import vibe.stream.wrapper;
 
 void render(scope HTTPServerRequest req, scope HTTPServerResponse res)
 {
-	auto dst = StreamOutputRange(res.bodyWriter);
+	auto dst = StreamOutputRange(res.bodyWriter.asInterface!OutputStream);
```

Comment:
This is to avoid this error:

```
source/app.d(8,30): Error: template vibe.stream.wrapper.StreamOutputRange cannot deduce function from argument types !()(InterfaceProxy!(OutputStream)), candidates are:
/home/seb/.dub/packages/vibe-d-0.8.3-alpha.2/vibe-d/stream/vibe/stream/wrapper.d(290,32):        vibe.stream.wrapper.StreamOutputRange()(OutputStream stream)
/home/seb/.dub/packages/vibe-d-0.8.3-alpha.2/vibe-d/stream/vibe/stream/wrapper.d(292,1):        vibe.stream.wrapper.StreamOutputRange(OutputStream, ulong buffer_size = 256) if (isOutputStream!OutputStream)
```

Is the `InterfaceProxy` really necessary? Looks like it creates a lot of problems with the upcoming 0.8.3

Reply:
I think `streamOutputRange(res.bodyWriter)` should work here (without the additional overhead of `asInterface`). Unfortunately `InterfaceProxy` is still necessary when using `vibe-core`, but eventually all uses of it will be replaced by static types using templates.

Reply:
I thought so too, but DMD tells me otherwise: https://travis-ci.org/rejectedsoftware/diet-ng/jobs/332745186

Reply:
Ah, note the small `s`. It's just a constructor wrapper to enabled IFTI.

Reply:
> Ah, note the small s

Note sure what you mean by this?

Reply:
It's `streamOutputRange` vs. `StreamOutputRange`. The former is the correct one and the latter is what appears to be called for the failing build output.

Reply:
Ah got it. Thanks!